### PR TITLE
adding irida-sistr-results recipe

### PR DESCRIPTION
**Pull request by tizzir**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Adds a recipe for irida-sistr-results (https://github.com/phac-nml/irida-sistr-results)

### CODE REVIEW

**Comment by tizzir** [comment]
The tests seem to fail because of a permission error on ```/opt/host-conda-bld/noarch```
Is this because of something I did?

When I manually alter the permissions to fix this on my machine simulate-travis runs the tests fine

**Comment by druvus** [comment]
Restarted the test

File: recipes/irida-sistr-results/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# install dependencies through pip
+pip install -r requirements.txt
```
Comment (druvus) [inline recipes/irida-sistr-results/build.sh]: Those should according bioconda's guidelines be added in the requirement section

**Comment by druvus** [comment]
@tizzir  Please see my inline comment about adding dependencies.

**Comment by tizzir** [comment]
Updated! Just waiting on my rauth package to be merged into conda-forge, then I'll push my new recipe. Thanks @druvus!

**Comment by tizzir** [comment]
Everything should be ready to be merged now, unless I missed anything :)

**Comment by tizzir** [comment]
Thanks @bgruening!

**Comment by cbrueffer** [comment]
Merged; thanks everyone!

**Pull request merged**